pkg/test/echo/docker/Dockerfile.app_sidecar_base_centos

```diff
@@ -1,7 +1,7 @@
 ARG VM_IMAGE_VERSION=8
 FROM centos:${VM_IMAGE_VERSION}
 
-# hadolint ignore=DL3005,DL3008
+# hadolint ignore=DL3005,DL3008,DL3031,DL3032,DL3033
```

Comment:
Should we actually fix the 3031 and 3032 errors?

Reply:
This is only used for the echo app that is only used in integration testing. This isn't for end users.  I can fix it if you'd like.

For DL3032 `yum clean all` is being run, but the linter doesn't pick up.

Reply:
I don't think the `yum clean all` is being run (missing &&).

Reply:
Fixed that issue in https://github.com/istio/istio/pull/30998